tests/filters/test_filters.py

```diff
@@ -172,6 +172,34 @@ def fruits(filters: FruitFilter) -> List[Fruit]:
     ]
 
 
+def test_resolver_filter_with_info(fruits):
+    from strawberry.types.info import Info
+
+    @strawberry_django.filters.filter(models.Fruit, lookups=True)
+    class FruitFilterWithInfo:
+        id: auto
+        name: auto
+        custom_field: bool
+
+        def filter_custom_field(self, queryset, info: Info):
+            # Test here is to prove that info can be passed properly
```

Comment:
Just to make sure it comes as `Info` properly and not with some other value

```suggestion
            # Test here is to prove that info can be passed properly
            assert isinstance(info, Info)
```